src/modules/addsaying.ts

```diff
@@ -6,14 +6,18 @@ export default ChatCommand({
   description: 'Create a new saying about myth mentions',
   options: new OptionBuilder().string('message', 'The saying to add', true),
   handle(options) {
-    Saying.create({ message: options.message })
-      .then(() => {
-        console.log(`Added new saying: "${options.message}"`);
-        this.reply(`Successfully added saying: "${options.message}"`);
-      })
-      .catch(() => {
-        console.error(`Problem adding saying: "$${options.message}"`);
-        this.reply(`Problem adding saying: "$${options.message}"`);
-      });
+    if (!this.member.roles.cache.has(process.env.ADMIN_PING_ROLE)) {
+      this.reply('Sorry, you don\'t have permission to add a saying!');
```

Comment:
i would like this to be an ephemeral reply by making it `.reply({ ephemeral: true, content: '...'  })`